components/_header.scss

```diff
@@ -39,3 +40,20 @@ header {
     display: none;
   }
 }
+
+.flash {
+  @extend h3;
+
+  background: $tws-yellow;
+  padding: .5em 1em;
```

Comment:
`.5` should be written with a leading zero as `0.5`<br>Properties should be ordered background, border, border-left, border-right, color, padding, text-align, width